azure-pipelines.yml

```diff
@@ -639,10 +639,10 @@ jobs:
 
        echo "== Test yum package on CentOS =="
 
-       docker pull centos:centos7
-       docker run --rm -v $SYSTEM_ARTIFACTSDIRECTORY/yum:/mnt/yum centos:centos7 /bin/bash -c "yum --nogpgcheck localinstall /mnt/yum/$YUM_NAME -y && time az self-test && time az --version && sleep 5"
+       docker pull centos:centos8
```

Comment:
what's stage of centos7 now? are we removing the support, why not test it now

Reply:
No, we support both centos 7 and centos 8. The default python for centos 7 is Python 2. It's more convenient to test on centos 8 when calling `scripts/ci/build.sh` which invokes `/usr/bin/python`.